### PR TITLE
endure has less crit hp, 150 from 200

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
crit hp from 200 to 150

## Why It's Good For The Game
i do not believe i have ever actually seen a ravager die from crit hp in endure, like ever.
Their ehp is still incredibly high considering this is plus your actual health and you have armor+passive regen.

## Changelog
:cl:
balance: endure has less crit hp, 150 from 200
/:cl:

### CODE REVIEW

**Comment by lKiseki** [comment]
No, this is an excessive nerf. 150 would be a more acceptable nerf.
MJP, seriously, stop doing kneejerk PRs like these.

**Comment by Pariah919** [comment]
>No, this is an excessive nerf. 150 or 175 would be more acceptable nerfs.
>
175 is still a lot more health and i doubt this will ever make ravagers die in reality but david told me to make it 175 so it'll be 175

**Comment by SilencedMP5A5** [comment]
doesn't seem too excessive to me tbh

maybe they'll actually die for once when being focused by entire ungaballs

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -527,7 +527,7 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 
 #define RAVAGER_ENDURE_DURATION				10 SECONDS
 #define RAVAGER_ENDURE_DURATION_WARNING		0.7
-#define RAVAGER_ENDURE_HP_LIMIT				-200
+#define RAVAGER_ENDURE_HP_LIMIT				-125
```
Comment (Pariah919) [inline code/__DEFINES/mobs.dm:530]:
```suggestion
#define RAVAGER_ENDURE_HP_LIMIT				-175
```

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -527,7 +527,7 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 
 #define RAVAGER_ENDURE_DURATION				10 SECONDS
 #define RAVAGER_ENDURE_DURATION_WARNING		0.7
-#define RAVAGER_ENDURE_HP_LIMIT				-200
+#define RAVAGER_ENDURE_HP_LIMIT				-175
```
Comment (Pariah919) [inline code/__DEFINES/mobs.dm:530]:
```suggestion
#define RAVAGER_ENDURE_HP_LIMIT				-150
```

**Comment by Al-1ce** [comment]
> No, this is an excessive nerf. 150 or 175 would be more acceptable nerfs.


I'd typically agree, but as a xeno main and someone who generally plays rav I can tell you it is near godmode. Ravagers really shouldn't be pushing or attacking with negative HP and this only encourages that behavior without any drawback or consequence.

**Comment by Carnage068** [comment]
> > No, this is an excessive nerf. 150 or 175 would be more acceptable nerfs.
> 
> I'd typically agree, but as a xeno main and someone who generally plays rav I can tell you it is near godmode. Ravagers really shouldn't be pushing or attacking with negative HP and this only encourages that behavior without any drawback or consequence.

Are you kidding me? Pushing or attacking with negative HP is fucking dangerous, dude. If a Rav wants to take that risk they should be allowed to do so, LOL. They're supposed to be a high risk high reward caste.

**Comment by Carnage068** [comment]
They don't usually die in Endure because they run away before they drop too low, that's the point. You're supposed to get out before you go beyond your breaking point and end up insta-dying. Really don't get why that's so hard to understand- Not to mention, Rav is already having trouble due to the stupid plasma changes. I guess people are just salty and want to see the caste gutted. Oh well.

**Comment by SplinterGP** [comment]
> 
> 
> They don't usually die in Endure because they run away before they drop too low, that's the point. You're supposed to get out before you go beyond your breaking point and end up insta-dying. Really don't get why that's so hard to understand- Not to mention, Rav is already having trouble due to the stupid plasma changes. I guess people are just salty and want to see the caste gutted. Oh well.
> 
> Oh yeah, not to mention that Endure has an entire 1 minute cooldown. Smh.

i mean you dont need to use it for that, if you use it right its just effectively more HP to run away since after it ends the damage that'd kill you doenst stay

**Comment by Carnage068** [comment]
> > They don't usually die in Endure because they run away before they drop too low, that's the point. You're supposed to get out before you go beyond your breaking point and end up insta-dying. Really don't get why that's so hard to understand- Not to mention, Rav is already having trouble due to the stupid plasma changes. I guess people are just salty and want to see the caste gutted. Oh well.
> > Oh yeah, not to mention that Endure has an entire 1 minute cooldown. Smh.
> 
> i mean you dont need to use it for that, if you use it right its just effectively more HP to run away since after it ends the damage that'd kill you doenst stay

Then you're using it as an escape, thus sacrificing its offensive potential. Are you saying you don't want people to make that choice in the first place?

**Comment by SplinterGP** [comment]
> Then you're using it as an escape, thus sacrificing its offensive potential. Are you saying you don't want people to make that choice in the first place?

its not sacrificing a offensive potential if you cant use it offensively at all in the situation

**Comment by Carnage068** [comment]
> its not sacrificing a offensive potential if you cant use it offensively at all in the situation

You can't use it offensively? If you're low on health while attacking someone that's also low on health, if you use Endure, you have slightly more time to finish them off before you have to retreat. The risk for this is that if you wait until you're really low on health before Enduring, you could very well get staggered before then, and you'll end up getting completely screwed over and most likely dying. Plus, by choosing to stay in longer with the ability, instead of using it to run, you open yourself up to the possibility of getting bursted down and insta-dying before you have the chance to get away. This isn't very common (nor should it be imo), but it DOES happen, and it's something you have to keep in mind when playing Rav.

Unless, that is, you mean to say that if the changes go through, Endure won't be able to be used offensively, which in that case I'd somewhat agree with you. You could still probably use it offensively, but these changes would encourage defensive use and discourage offensive use, which I really, REALLY don't find fitting for a caste that's supposed to be all about offense.

**Comment by SplinterGP** [comment]
> 
> 
> > its not sacrificing a offensive potential if you cant use it offensively at all in the situation
> 
> You can't use it offensively? If you're low on health while attacking someone that's also low on health, if you use Endure, you have slightly more time to finish them off before you have to retreat. The risk for this is that if you wait until you're really low on health before Enduring, you could very well get staggered before then, and you'll end up getting completely screwed over and most likely dying. Plus, by choosing to stay in longer with the ability, instead of using it to run, you open yourself up to the possibility of getting bursted down and insta-dying before you have the chance to get away. This isn't very common (nor should it be imo), but it DOES happen, and it's something you have to keep in mind when playing Rav.
> 
> Unless, that is, you mean to say that if the changes go through, Endure won't be able to be used offensively, which in that case I'd somewhat agree with you. You could still probably use it offensively, but these changes would encourage defensive use and discourage offensive use, which I really, REALLY don't find fitting for a caste that's supposed to be all about offense.

i mean that not always you can use it offensively for N reasons of it not being able to, such as marines coming to help, in a case where using it defensively is 100% the right choice, and generally it lasts 10 to 15 seconds, which means its 1 minute cooldown is actually 45, then you afterwards are likely spending time resting, which then makes the cooldown not that big at all.

**Comment by Carnage068** [comment]
> i mean that not always you can use it offensively for N reasons of it not being able to, such as marines coming to help, in a case where using it defensively is 100% the right choice, and generally it lasts 10 to 15 seconds, which means its 1 minute cooldown is actually 45, then you afterwards are likely spending time resting, which then makes the cooldown not that big at all.

Marines coming to help? You mean like, getting swarmed by a huge unga ball? Well yeah, that's true- But if you're using it defensively in that situation, the 50 HP nerf from this PR won't really change anything. The vast majority of the time the Rav is still going to get away and live. The only thing this really hurts is offensive play, which is what Ravager is supposed to be all about. Also, by the time you've finished recovering your health back to full, Endure is still going to be on cooldown for anywhere from 40 to 15 seconds, and the fact that you'll have to wait that long before you're able to go in with the safety of the ability is significant. All this PR is going to do is push Rav in a direction that doesn't make sense for it.

**Comment by SplinterGP** [comment]
> > i mean that not always you can use it offensively for N reasons of it not being able to, such as marines coming to help, in a case where using it defensively is 100% the right choice, and generally it lasts 10 to 15 seconds, which means its 1 minute cooldown is actually 45, then you afterwards are likely spending time resting, which then makes the cooldown not that big at all.
> 
> Marines coming to help? You mean like, getting swarmed by a huge unga ball? Well yeah, that's true- But if you're using it defensively in that situation, the 50 HP nerf from this PR won't really change anything. The vast majority of the time the Rav is still going to get away and live. The only thing this really hurts is offensive play, which is what Ravager is supposed to be all about. Also, by the time you've finished recovering your health back to full, Endure is still going to be on cooldown for anywhere from 40 to 15 seconds, and the fact that you'll have to wait that long before you're able to go in with the safety of the ability is significant. All this PR is going to do is push Rav in a direction that doesn't make sense for it.

Dude 45/15 seconds is nothing, youre not always going to go overcommiting, saying thats a big cooldown is too much of a stretch. It being 150 or 200 crit hp is already 150/200 extra HP since otherwise youd be on floor in crit.

**Comment by Carnage068** [comment]
> Dude 45/15 seconds is nothing, youre not always going to go overcommiting, saying thats a big cooldown is too much of a stretch. It being 150 or 200 crit hp is already 150/200 extra HP since otherwise youd be on floor in crit.

Yes, and unlike regular HP, endure HP goes away after 10 seconds, meaning if you overstay your welcome, or get surprised attack after you've lost it, you're pretty much fucked. Also, how the hell is 45 seconds nothing? That's almost a minute of waiting time to use your safest ability. Not to mention, thanks to your changes, it's impossible to use Eviscerating Charge now, so basically, most of the time Rav only has one ability. I don't know- That just doesn't sound like good game design or balance to me. You do you, I suppose.

**Comment by SplinterGP** [comment]
People have been liking ravager as it is now, seems pretty much a you issue to me, you cant have everything

**Comment by Carnage068** [comment]
> People have been liking ravager as it is now, seems pretty much a you issue to me, you cant have everything



> People have been liking ravager as it is now, seems pretty much a you issue to me, you cant have everything

Then just finish the job and remove Eviscerating Charge from the game, so it won't be a noob trap anymore.

**Comment by SplinterGP** [comment]
Lol

**Comment by TiviPlus** [comment]
Im going to combine this with the leech pr to move it towards rage insteadof endure

**Review by TiviPlus — APPROVED** [review]

**Comment by Carnage068** [comment]
> Im going to combine this with the leech pr to move it towards rage insteadof endure

Thank fucking god. That actually seems like a good change, for once.

**Pull request merged**